components/service-operator/api/v1beta1/postgres_types.go

```diff
@@ -24,6 +24,7 @@ import (
 
 // AWS allows specifying configuration for the Postgres RDS instance
 type AWS struct {
+	DiskSize     int    `json:"diskSize,omitempty"`
```

Comment:
@blairboy362 has just been annoying in real life :smile: and suggested that `diskSize` isn't AWS-specific. I'm of the opinion that we can change this later when we make `local` or whatever be a thing, or when we actually implement this to be a working thing (ie the next story)?

Reply:
If this is an integer, can the unit be in the property name?
```suggestion
	DiskSizeGb     int    `json:"diskSizeGb,omitempty"`
```

Reply:
Sure. Applied.